quickstarts/microsoft.signalrservice/signalr/main.bicep

```diff
@@ -0,0 +1,121 @@
+@description('The globally unique name of the SignalR resource to create.')
+param name string = uniqueString(resourceGroup().id)
+
+@description('Location for the SignalR resource.')
+param location string = resourceGroup().location
+
+@description('The pricing tier of the SignalR resource.')
+@allowed([
+  'Free_F1'
+  'Standard_S1'
+])
+param pricingTier string = 'Standard_S1'
+
+@description('The number of SignalR Unit.')
+@allowed([
+  1
+  2
+  5
+  10
+  20
+  50
+  100
+])
+param capacity int = 1
+
+@description('Visit https://github.com/Azure/azure-signalr/blob/dev/docs/faq.md#service-mode to understand SignalR Service Mode.')
+@allowed([
+  'Default'
+  'Serverless'
+  'Classic'
+])
+param serviceMode string = 'Default'
+
+@allowed([
+  'true'
+  'false'
+])
+param enableConnectivityLogs string = 'true'
```

Comment:
let use bool as the type for all the true/false params (and remove allowedValues)